Makefile

```diff
@@ -46,6 +46,9 @@ BUILD_VM_SRC_amd64=https://cloud-images.ubuntu.com/focal/current/focal-server-cl
 BUILD_VM_SRC=$(BUILD_VM_SRC_$(ZARCH))
 
 UNAME_S := $(shell uname -s)
+HOSTARCH:=$(subst aarch64,arm64,$(subst x86_64,amd64,$(shell uname -m)))
+
+LINUXKIT_HOST_TARGET=$(HOSTARCH)/$(shell uname -s | tr '[A-Z]' '[a-z]')
```

Comment:
Is this reversed? It looks like this gives `amd64/linux` or `arm64/darwin` as opposed to `linux/amd64` or `darwin/arm64`

Reply:
Thanks for spotting that! You're right, that the Welcome message does print "_EVE will be built for the host platform arm64/darwin, by default_", so I've fixed that, now.

Reply:
Should be fixed, in fd41f195a1d16cb442a71f281bbee0e5d9d107bd